.travis.yml

```diff
@@ -1,29 +1,45 @@
+sudo: required
```

Comment:
It switches to some distro where sudo is provided, it just happened to be trusty at the moment. If you want trusty (and you do for that apt line), it's better to specify it.